testing/smoke-ide-test/build.gradle.kts

```diff
@@ -45,13 +42,47 @@ plugins.withType<IdeaPlugin> {
     }
 }
 
-tasks.register<SmokeIdeTest>("smokeIdeTest") {
-    group = "Verification"
-    maxParallelForks = 1
-    systemProperties["org.gradle.integtest.executer"] = "forking"
-    testClassesDirs = smokeIdeTestSourceSet.output.classesDirs
-    classpath = smokeIdeTestSourceSet.runtimeClasspath
-    dependsOn(unzipIdeStarter)
+tasks {
+    val unzipIdeStarter by registering(Sync::class) {
+        from(zipTree(ideStarter.elements.map { it.single() }))
+        into(layout.buildDirectory.dir("ideStarter"))
+    }
+
+    val fetchGradle by registering(RemoteProject::class) {
+        remoteUri = rootDir.absolutePath
+        ref = buildCommitId
+    }
+
+    val shrinkGradle by registering(Sync::class) {
```

Comment:
We're shrinking `gradle/gradle` project as we don't need source code files there, we need only build logic. This is the similar approach as `Gradleception Smoke Test` does.